packages/ember-metal/lib/meta.js

```diff
@@ -46,6 +47,24 @@ let members = {
   tag: ownCustomObject
 };
 
+// This is less than ideal, but `runInDebug` is
+// not populated by the time we need it here. So
+// we create a function that has the same signature
+// as the real `runInDebug` so that it is stripped
+// from the production build. Unfornately, this
+// function will remain in production build.
+let runInDebug = (fn) => fn();
```

Comment:
This confuses me.  When you say, it isn't populated, do you mean that it doesn't run the callback or that the import is undefined?